drivers/gpio/gpio_mcux.c

```diff
@@ -294,7 +294,7 @@ static uint32_t get_port_pcr_irqc_value_from_flags(const struct device *dev, uin
 
 #if (defined(FSL_FEATURE_GPIO_HAS_INTERRUPT_CHANNEL_SELECT) && \
 		FSL_FEATURE_GPIO_HAS_INTERRUPT_CHANNEL_SELECT) \
-        ||  (defined(FSL_FEATURE_PORT_HAS_NO_INTERRUPT) && FSL_FEATURE_PORT_HAS_NO_INTERRUPT)
+		||  (defined(FSL_FEATURE_PORT_HAS_NO_INTERRUPT) && FSL_FEATURE_PORT_HAS_NO_INTERRUPT)
 
```

Comment:
@robertinant with Zephyr PRs you don't need to add a commit to fix the previous commits CI errors. Just fixup the original commit and force push your branch.